web/includes/Monitor.php

```diff
@@ -268,7 +268,12 @@ public static function getStatuses() {
     'ArchivedEvents' =>  array('type'=>'integer', 'default'=>null, 'do_not_update'=>1),
     'ArchivedEventDiskSpace' =>  array('type'=>'integer', 'default'=>null, 'do_not_update'=>1),
   );
-
+  public function Janus_Pin() {
+    $cmd = getZmuCommand(' --janus-pin -m '.$this->{'Id'});
```

Comment:
Doesn't have to be fixed now, but this misses the multi-server case, in which we would have to use the API, so the API needs support for Janus_Pin too.